File: Patterns/js/pattern10.js
/*
n = 4

4 4 4 4
4 4 4 4
4 4 4 4
4 4 4 4

*/

function pattern(n) {
  let ans = "";
  let row = 1;

  while (row <= n) {
    let col = 1;

    while (col <= n) {
      ans += n + " ";
      col++;
    }

    ans += "\n";
    row++;
  }

  console.log(ans);
}

pattern(4);
pattern(10);

File: Patterns/go/pattern10.go
package main

import "fmt"

/*

n = 4

A B C D
B C D E
C D E F
D E F G


*/

func main() {
	var n int
	fmt.Scan(&n)

	row := 1
	initialValue := 'A'

	for row <= n {
		col := 1
		value := initialValue
		for col <= n {
			fmt.Print(string(value), " ")
			value++
			col++
		}
		fmt.Print("\n")
		initialValue++
		row++
	}
}